include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -11,34 +11,34 @@
 #    endif
 
 // Base classes.
-#    include <alpaka/atomic/AtomicCpu.hpp>
-#    include <alpaka/atomic/AtomicHierarchy.hpp>
-#    include <alpaka/atomic/AtomicNoOp.hpp>
-#    include <alpaka/atomic/AtomicOmpBuiltIn.hpp>
-#    include <alpaka/block/shared/dyn/BlockSharedMemDynMember.hpp>
-#    include <alpaka/block/shared/st/BlockSharedMemStMember.hpp>
-#    include <alpaka/block/sync/BlockSyncNoOp.hpp>
-#    include <alpaka/core/DemangleTypeNames.hpp>
-#    include <alpaka/idx/bt/IdxBtZero.hpp>
-#    include <alpaka/idx/gb/IdxGbRef.hpp>
-#    include <alpaka/intrinsic/IntrinsicCpu.hpp>
-#    include <alpaka/math/MathStdLib.hpp>
-#    include <alpaka/mem/fence/MemFenceOmp2Blocks.hpp>
-#    include <alpaka/rand/RandStdLib.hpp>
-#    include <alpaka/warp/WarpSingleThread.hpp>
-#    include <alpaka/workdiv/WorkDivMembers.hpp>
+#    include "alpaka/atomic/AtomicCpu.hpp"
```

Comment:
@bernhardmgruber Could you please push the first commit as tools author: 

```bash
GIT_AUTHOR_NAME="Tools" GIT_AUTHOR_EMAIL="alpaka@hzdr.de" git commit ...
```

Reply:
Done